BuildChecker/BuildChecker.csproj

```diff
@@ -49,4 +51,4 @@ https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild
   </Target>
   <Target Name="CoreCompile">
   </Target>
-</Project>
+</Project>
```

Comment:
im not a maints but untouch probably

Reply:
Imma be honest I dunno what it means to untouch

Reply:
> it means to remove the change you did there.

I never even touched that file is why I'm confused :blunt:

Reply:
Giving a hand, i don't know git properly, what i do in these cases is do the inverse of the changes here. it seems it removed a end of line there, try doing a reset to it's previous commit.